Prepare release 4.9.0

k0ekk0ek opened the pull request.

Commit:
Set up release notes for upcoming release

Commit:
Fix timing sensitivity in ixfr_outsync test

wcawijngaards reviewed (approved):
Looks good! That could make the release.

Commit:
Test if debug is available in do-tests

Commit:
Force timeout from NSD in ixfr_gone test

read command in OpenBSD does not support -t (timeout). Force a timeout
from NSD instead.

Commit:
Update expressions in ixfr_and_restart test

Update regular expressions so they work on OpenBSD too.

Commit:
Make algorithm explicit in control-repattern test

Commit:
Switch algorithm to hmac-256 for testplan_mess

k0ekk0ek commented:
Tests now all passing on Linux, FreeBSD and OpenBSD
